optimum/gptq/quantizer.py

```diff
@@ -71,6 +71,7 @@ def __init__(
         pad_token_id: Optional[int] = None,
         disable_exllama: bool = False,
         max_input_length: Optional[int] = None,
+        cache_block_outputs: Optional[bool] = False,
```

Comment:
I would prefer to leave it True to not break BC.

Reply:
Fixed